packages/sdk-components-react/src/head-slot.tsx

```diff
@@ -0,0 +1,78 @@
+import {
+  getClosestInstance,
+  ReactSdkContext,
+  type Hook,
+} from "@webstudio-is/react-sdk/runtime";
+import {
+  forwardRef,
+  type ElementRef,
+  type ComponentProps,
+  useContext,
+} from "react";
+import { XmlNode } from "./xml-node";
+
+export const defaultTag = "head";
+
+export const HeadSlot = forwardRef<
+  ElementRef<"div">,
+  { "data-ws-expand"?: boolean } & Omit<
+    ComponentProps<typeof defaultTag>,
```

Comment:
I don't think we actually support any props in head slot and should not generate them

Reply:
Does not used anywhere on prod, just for hooks

Reply:
Done